test/integration/webhookgithub_test.go

```diff
@@ -327,6 +327,42 @@ func mockBuildConfigImageParms(imageName, imageStream, imageTag string) *buildap
 						Secret: "secret102",
 					},
 				},
+				{
+					Type: buildapi.GitLabWebHookBuildTriggerType,
+					GitLabWebHook: &buildapi.WebHookTrigger{
```

Comment:
maybe change the name of this file if it is going to include gitlab and bitbucket tests also?